query-service/src/integrationTest/java/org/hypertrace/core/query/service/QueryServiceTestUtils.java

```diff
@@ -33,40 +35,69 @@ public static Filter createFilter(
     return Filter.newBuilder().setLhs(lhs).setOperator(op).setRhs(rhs).build();
   }
 
-  public static Function.Builder createTimeColumnGroupByFunction(
+  public static Expression createTimeColumnGroupByFunctionExpression(
       String timeColumn, String periodSecs) {
-    return Function.newBuilder()
-        .setFunctionName("dateTimeConvert")
-        .addArguments(
-            Expression.newBuilder()
-                .setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName(timeColumn)))
-        .addArguments(
-            Expression.newBuilder()
-                .setLiteral(
-                    LiteralConstant.newBuilder()
-                        .setValue(Value.newBuilder().setString("1:MILLISECONDS:EPOCH"))))
-        .addArguments(
-            Expression.newBuilder()
-                .setLiteral(
-                    LiteralConstant.newBuilder()
-                        .setValue(Value.newBuilder().setString("1:MILLISECONDS:EPOCH"))))
-        .addArguments(
-            Expression.newBuilder()
-                .setLiteral(
-                    LiteralConstant.newBuilder()
-                        .setValue(Value.newBuilder().setString(periodSecs))));
+    return Expression.newBuilder()
+        .setFunction(
+            Function.newBuilder()
+                .setFunctionName("dateTimeConvert")
+                .addArguments(
+                    Expression.newBuilder()
+                        .setColumnIdentifier(
+                            ColumnIdentifier.newBuilder().setColumnName(timeColumn)))
+                .addArguments(
+                    Expression.newBuilder()
+                        .setLiteral(
+                            LiteralConstant.newBuilder()
+                                .setValue(Value.newBuilder().setString("1:MILLISECONDS:EPOCH"))))
+                .addArguments(
+                    Expression.newBuilder()
+                        .setLiteral(
+                            LiteralConstant.newBuilder()
+                                .setValue(Value.newBuilder().setString("1:MILLISECONDS:EPOCH"))))
+                .addArguments(
+                    Expression.newBuilder()
+                        .setLiteral(
+                            LiteralConstant.newBuilder()
+                                .setValue(Value.newBuilder().setString(periodSecs))))
+                .build())
+        .build();
+  }
+
+  public static Expression createColumnExpression(String columnName) {
+    return Expression.newBuilder()
+        .setColumnIdentifier(ColumnIdentifier.newBuilder().setColumnName(columnName))
+        .build();
   }
 
-  public static ColumnIdentifier createColumnIdentifier(String columnName) {
-    return ColumnIdentifier.newBuilder().setColumnName(columnName).build();
+  public static Expression createFunctionExpression(String functionName, Expression columnNameArg) {
```

Comment:
naming - this is just an argument 
```suggestion
  public static Expression createFunctionExpression(String functionName, Expression argumentExpression) {
```